source/plugins/ruby/filter_cadvisor_health_container.rb

```diff
@@ -20,6 +20,7 @@ class CAdvisor2ContainerHealthFilter < Filter
         @@object_name_k8s_container = 'K8SContainer'
         @@counter_name_cpu = 'cpuusagenanocores'
         @@counter_name_memory_rss = 'memoryrssbytes'
+        @@cluster_health_model_enabled = HealthMonitorUtils.is_cluster_health_model_enabled
```

Comment:
I am assuming this function already exists in HM Utils from private preview as i dont see it in this PR ?